docs/specviz/export_data.rst

```diff
@@ -14,28 +14,38 @@ those data currently back into your Jupyter notebook:
 
     specviz.get_spectra()
 
-which yields a `specutils.Spectrum1D` object that you can manipulate however
-you wish.  You can then load the modified spectrum back into the notebook via
-the API described in :ref:`specviz-import-api`.
+which yields a either a single `specutils.Spectrum1D` object or a dictionary of 
+`specutils.Spectrum1D` (if there are multiple displayed spectra) that you can
+manipulate however you wish.  You can then load the modified spectrum back into
+the notebook via the API described in :ref:`specviz-import-api`.
 
 Alternatively, if you want more control over Specviz, you can access it the
-via the lower-level application interface that connects to the ``glue-jupyter``
-application level.  This is accessed via the ``.app`` attribute of the
-:py:class:`~jdaviz.configs.specviz.helper.Specviz` helper class.  For example:
+via the ``get_data`` method of the
+:py:class:`~jdaviz.configs.specviz.helper.Specviz` helper class. This method always
+returns a single spectrum; if there are multiple spectra loaded you must supply a
+label to the ``data_label`` argument. For example:
 
 .. code-block:: python
 
-    specviz.app.get_data_from_viewer('spectrum-viewer')
+    specviz.get_data(data_label='Spectrum 1')
 
-To extract a specific spectral subset:
+To extract a spectrum with a spectral subset applied:
 
 .. code-block:: python
 
-    specviz.app.get_data_from_viewer('spectrum-viewer', 'Subset 1')
+    specviz.get_data(subset_to_apply='Subset 1')
 
-For more on what you can do with this lower-level object, see the API sections
-and the
-`glue-jupyter documentation <https://glue-jupyter.readthedocs.io/en/latest/>`_.
+In this case, the returned `specutils.Spectrum1D` object will have a ``mask``
+attribute, where ``True`` corresponds to the region outside the selected subset
+(i.e., the region that has been masked out). You could load back in a copy of the
+spectrum containing only your subset by running:
+
+.. code-block:: python
+
+    spec = specviz.get_data(subset_to_apply='Subset 1')
+    subset_spec = Spectrum1D(flux=spec.flux[~spec.mask],
+                             spectral_axis=spec.spectral_axis[~spec.mask])
+    specviz.load_spectrum(subset_spec)
```

Comment:
I think this makes sense for now. But since `Spectrum1D` already has a mask attribute, we could(/should?) make `specviz.load_spectrum` load the mask so that you don't have to also index by the not-mask. Does the parser do this already/is this step necessary?

Reply:
The parser does load the mask (I believe you can show the mask and uncertainties, should be a toggle somewhere), but what Cami wanted to do was load in the data such that the are outside the subset is gone completely, which is what this does.